src/components/MainControls.tsx

```diff
@@ -58,6 +58,18 @@ export const MainControls: React.FC<{fullscreen: FullScreenHandle}> = ({fullscre
             })
           }
         />
+        <Button
+          icon='bg3'
+          highlighted={currentAmbianceCategoryName === 'bg3'}
+          onClick={
+            logEventClickWrapper({
+              onClick: () => handleCategoryChanger('bg3'),
+              eventData: {
+                actionId: 'bg3Category'
+              }
+            })
+          }
+        />
```

Comment:
Out of scope for this PR, but would be cool if we could update this entire React component to dynamically build the list of icons based on the config files instead of needing to manually add a new <Button /> for each new category. Something for a future improvement